packages/utils_browser/src/itownsUtil.js

```diff
@@ -5,10 +5,30 @@ const itowns = require('itowns');
 
 /** CAMERA */
 
+/**
+ * Compute a distance automatically for travel features of itowns Planar View
+ *
+ * @param {itowns.PlanarControls} controls - planar controls of itownsView
+ * @param {THREE.Vector3} targetPos - position of our target
+ * @returns {number} - the duration of travel to focus a targetPosition
```

Comment:
Maybe add a reference to the original (inaccessible) code within iTowns ?
https://github.com/iTowns/itowns/blob/v2.41.0/src/Controls/PlanarControls.js